src/main/java/net/dzikoysk/funnyguilds/basic/util/RegionUtils.java

```diff
@@ -55,18 +55,21 @@ public static boolean isNear(Location center) {
         PluginConfig s = Settings.getConfig();
         int i = s.regionSize;
         if (s.enlargeItems != null) {
-            i += s.enlargeItems.size() * s.enlargeSize;
+            i += (s.enlargeItems.size() * s.enlargeSize);
         }
-        int requiredDistance = 2 * i + s.regionMinDistance;
+        int requiredDistance = (2 * i) + s.regionMinDistance;
         for (Region region : regions) {
+            if (region.getCenter().equals(center)) continue;
```

Comment:
A jeżeli okaże się, że `getCenter()` zwraca nulla? Niżej jest warunek, który to sprawdza.

To jest drobna różnica pomiędzy #503, a tym commitem.